src/utils/components/SSHAccess/components/SSHCommand.tsx

```diff
@@ -82,7 +85,11 @@ const SSHCommand: React.FC<SSHCommandProps> = ({
       <DescriptionListDescription>
         <Stack hasGutter>
           <StackItem>
-            <SSHCheckbox sshServiceRunning={!!sshService} setSSHServiceRunning={onSSHChange} />
+            <SSHCheckbox
+              sshServiceRunning={Boolean(sshService)}
```

Comment:
Just a small FYI, you could have just left it as `!!sshService`. The `!!` operator converts any variable to a boolean value.

Reply:
Yes, we could use `!!`, but then _DeepSource_ is failing and protesting against using it...